src/main/java/com/alkemy/ong/dto/request/slide/SlideUpdateDto.java

```diff
@@ -0,0 +1,12 @@
+package com.alkemy.ong.dto.request.slide;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SlideUpdateDto {
```

Comment:
It could be called SlideResponseDto to keep a pattern in the nomenclature.

Reply:
It is a request DTO, and there is already a creation DTO called SlideRequestDto.

Reply:
Likewise, I consider the suggested change correct, so I refactor the name of SlideDto to SlideResponseDto. Thank you!